pkg/dao/interfaces.go

```diff
@@ -110,7 +111,11 @@ type SnapshotDao interface {
 	List(ctx context.Context, orgID string, repoConfigUuid string, paginationData api.PaginationData, filterData api.FilterData) (api.SnapshotCollectionResponse, int64, error)
 	ListByTemplate(ctx context.Context, orgID string, template api.TemplateResponse, repositorySearch string, paginationData api.PaginationData) (api.SnapshotCollectionResponse, int64, error)
 	FetchForRepoConfigUUID(ctx context.Context, repoConfigUUID string) ([]models.Snapshot, error)
+	FetchUnscoped(ctx context.Context, uuid string) (models.Snapshot, error)
```

Comment:
Small thing, but I think this should be renamed to "FetchModel" and use a boolean to determined scoped or unscoped. This would bring it more in line with what we do for other dao objects like Template.

Reply:
sure, will do 🫡